HMCL/src/main/resources/assets/lang/I18N_zh_CN.properties

```
@@ -1255,9 +1252,12 @@ terracotta.status.scanning.back=这将同时停止扫描局域网世界。
 terracotta.status.host_starting=正在启动房间
 terracotta.status.host_starting.back=这将会取消创建房间。
 terracotta.status.host_ok=已启动房间
-terracotta.status.host_ok.code=邀请码 (已自动复制到剪贴板)
+terracotta.status.host_ok.code=邀请码（已自动复制到剪贴板）
 terracotta.status.host_ok.code.copy=复制邀请码
-terracotta.status.host_ok.code.desc=请提醒您的朋友在 HMCL 或 PCL CE 多人联机功能中选择房客模式，并输入该邀请码。
+terracotta.status.host_ok.code.desc=提醒您的朋友们，请根据他们使用的启动器选择正确的选项，然后输入邀请码：\n\
```

这句话有些诡异，但下面这个建议也有些问题

```suggestion
terracotta.status.host_ok.code.desc=请提醒您的朋友按下列步骤加入房间：\n\
```